zdm/tests/mc_statistics.py

```diff
@@ -0,0 +1,386 @@
+
```

Comment:
I might move this to a new folder named `mcmc/`

Typically, `tests` is reserved for "unit tests", i.e. scripts
that test functionality of bits of the code.  Although I realize
that the MCMC codes are in part for that function. :)

Reply:
Well, this is currently only a mc, rather than a mcmc... but please feel free!

Reply:
I am deciding to leave this in tests for now, it is a relatively small-scale program, and makes sense as a 'test' - it is absolutely NOT a full MCMC!